GraphQL.Server.sln

```diff
@@ -15,7 +15,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".Solution Items", ".Solutio
 		.gitattributes = .gitattributes
 		.gitignore = .gitignore
 		Directory.Build.props = Directory.Build.props
-		LICENSE.md = LICENSE.md
```

Comment:
I think that this file is no longer necessary, because we are using `PackageLicenseExpression` although I still have doubts, because the license for the repository as a whole has nothing to do with specific tags in csproj. In any case, we need to regenerate the new license file on behalf of the GraphQL org